map/src/components/add-instructions.tsx

```diff
@@ -1,70 +1,582 @@
+                        />
+                      </label>
+                    </label>
+                  </li>
+                </ul>
+                <label htmlFor={htmlFor}>
+                  Optional long-description of what the project does or aims to
+                  do
+                  <input
+                    type="text"
+                    name="description"
+                    placeholder="Your answer"
+                    onChange={this.handleInputChange}
+                  />
+                </label>
+                <label htmlFor={htmlFor}>
+                  Website URL?
+                  <input
+                    type="text"
+                    name="website"
+                    placeholder="Your answer"
+                    onChange={this.handleInputChange}
+                  />
+                </label>
+                <label htmlFor={htmlFor}>
+                  How do people that require help get in touch?
+                  <input
+                    type="text"
+                    name="instructions"
+                    placeholder="Your answer"
+                    onChange={this.handleInputChange}
+                  />
+                </label>
+                <label htmlFor={htmlFor}>
+                  How do people that want to volunteer get in touch?
+                  <input
+                    type="text"
+                    name="volunteerInstructions"
+                    placeholder="Your answer"
+                    onChange={this.handleInputChange}
+                  />
+                </label>
+                <label htmlFor={htmlFor}>
+                  Any other URLs / Facebook Groups / Contact details?
+                  <input
+                    type="text"
+                    name="otherContact"
+                    placeholder="Your answer"
+                    onChange={this.handleInputChange}
+                  />
+                </label>
+              </form>
+              <footer>
+                <button
+                  type="button"
+                  className="next-button"
+                  onClick={this.submit}
+                >
+                  Submit
+                </button>
+                <button
+                  type="button"
+                  className="prev-button"
+                  onClick={() => setAddInfoStep('set-radius')}
+                >
+                  <MdChevronLeft />
+                  Back
+                </button>
+              </footer>
+            </>
+          )}
+          {addInfoStep === 'farewell' && (
+            <>
+              <p>Final State</p>
+              <table>
+                <thead>
+                  <tr>
+                    <td>variable</td>
+                    <td>value</td>
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr>
+                    <td>center</td>
+                    <td>{JSON.stringify(center)}</td>
```

Comment:
I don't think is very good idea to use json stringify to show data to the user.

I'ts safer to put `lat: {center.lat}, long: {center.lng}`

Reply:
Yes, it's right! I've put them just for experimentation purposes. I have no idea yet for the last step screen. Just planning to put some information messages such as "Thank you!  We'll review your application soon". Maybe we can put the data posted but not that JSON way 😄

Reply:
I think just the message you said is fine for now. If really necessary then we can show a table with the inserted data like you have in the screenshot but more formatted.